cjworkbench/i18n/trans.py

```diff
@@ -124,6 +128,10 @@ def for_module_zipfile(cls, module_zipfile: ModuleZipfile):
                     f"Invalid po file for module {module_zipfile.module_id_and_version} in locale {locale_id}: {err}"
                 )
                 catalogs[locale_id] = Catalog()
+            except KeyError as err:
+                pass
```

Comment:
Why is this `pass`, while `PoFileError` leads to `Catalog()`?

I'd expect the two cases to lead to the same result (though only `PoFileError` would cause logging).